Document Hash::Util::hash_value($string, $seed)

yecril71pl opened the pull request:
Returns the hash value as if computed with a different seed.

yecril71pl committed:
Document Hash::Util::hash_value($string, $seed)

Returns the hash value as if computed with a different seed.

jkeenan commented on ext/Hash-Util/lib/Hash/Util.pm:
```diff
@@ -509,13 +509,19 @@ Perl has been built with. Possible sizes may be but are not limited to
+    my $hash_value = hash_value($string, $seed);
+
+hash_value($string)
+returns the current perl's internal hash value for a given string.
```
POD formatting recommendation:  Enclose `hash_value($string)` in C<...> code formatting.

jkeenan commented on ext/Hash-Util/lib/Hash/Util.pm:
```diff
@@ -509,13 +509,19 @@ Perl has been built with. Possible sizes may be but are not limited to
+
+hash_value($string)
+returns the current perl's internal hash value for a given string.
+hash_value($string, $seed)
```
Same here.

jkeenan commented on ext/Hash-Util/lib/Hash/Util.pm:
```diff
@@ -509,13 +509,19 @@ Perl has been built with. Possible sizes may be but are not limited to
+hash_value($string, $seed)
+returns the hash value as if computed with a different seed.
+If the custom seed is too short, the function errors out.
+The minimum length of the seed is implementation-dependent.
```
In your text editor, consider reformatting paragraphs so that they wrap to 72 characters/line.

jkeenan commented on ext/Hash-Util/lib/Hash/Util.pm:
```diff
@@ -509,13 +509,19 @@ Perl has been built with. Possible sizes may be but are not limited to
+It may be different
+depending on invocation, environment variables, perl version,
 architectures, and build options.
 
```
I'll let others comment on the correctness of these documentation changes.

yecril71pl committed:
respect line width, mark C<hash_value($string)>

I am not responsible for applying the style recommendations to the whole document; it should be done in a separate commit.

yecril71pl committed:
fix author test

yecril71pl committed:
fix version test

khwilliamson commented:
Merged via  #19436
